cloudevents/tests/test_http_events.py

```diff
@@ -110,13 +113,13 @@ def test_emit_binary_event(specversion):
         "ce-source": "<event-source>",
         "ce-type": "cloudevent.event.type",
         "ce-specversion": specversion,
-        "Content-Type": "application/cloudevents+json"
+        "Content-Type": "text/plain"
     }
-    event = CloudEvent(test_data, headers=headers)
+    data = json.dumps(test_data)
```

Comment:
I thought that we were testing the sender side in `test_binary_to_request` on line 236. My concern here was that there could be a reciprocal bug in sending and receiving (for example, if the receive did not strip the `ce-` prefix from attributes, and then the response encoding did not add `ce-`, it would be impossible to tell if we use the library on both paths).

Reply:
Fair point. We can leave this as it is and in a future PR I'll just add an additional test verifying  cloudevents on both ends work just to ensure the sdk can talk to itself.

Reply:
I actually ended up adding a loop through the SDK in `test_roundtrip_non_json_event`.